ui/src/app/accusoft.component.js

```diff
@@ -101,43 +106,69 @@ export default function Accusoft(props) {
         'Content-Type': 'application/json'
       },
     }).then(res => res.json())
-      .then(user => setUser(user))
+      .then(data => setUser(data.user))
+  }
+
+  function deleteAll() {
+    fetch(`http://localhost:3001/delete-all`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      }
+    }).then(res => res.json())
+      .then(() => setDocList([]))
+  }
+
+  function deleteDoc(id) {
+    fetch('http://localhost:3001/doc', {
+      method: 'DELETE',
+      body: JSON.stringify({ id }),
+      headers: {
+        'Content-Type': 'application/json'
+      }
+    }).then(res => res.ok && setDocList(docList => docList.filter(doc => doc.rowid !== id)))
   }
 
   function insertText(name) {
     editor.insertText({ text: name });
   }
 
   return (
-    <>
```

Comment:
just rearranging things and adding delete buttons 
![image](https://user-images.githubusercontent.com/9112811/77451141-e5ccc100-6db9-11ea-9f8e-202e10a9156b.png)